app/views/mailer/gem_pushed.html.erb

```diff
@@ -0,0 +1,33 @@
+<p>
+  A gem you are responsible for was recently pushed to RubyGems.org.
```

Comment:
:thinking: I think this should be rephrased a little. I'm not feeling "responsible" for some of my pushed gems, but I'm owner of those gems for example. Maybe we can write it more tlike "...gem you have push access to have been recently pushed...".

Reply:
Thanks for the feedback @simi. Is it OK to release it like this for now and address this in a later PR?

Reply:
Sure if it will not be deployed before the text will get fixed. It will be great to start this feature with consistent email template. It can be suspicious to get this email template changed often IMHO.